hypha/apply/determinations/forms.py

```diff
@@ -472,6 +475,12 @@ def clean(self):
 
         return cleaned_data
 
+    def clean_outcome(self):
+        if not self.cleaned_data['outcome']:
+            self.add_error('outcome', 'Outcome is a required field')
+            return self.cleaned_data
+        return int(self.cleaned_data['outcome'])
```

Comment:
I guess this wouldn't be required if we do not remove the default from model field?